game/main.py

```diff
@@ -22,13 +22,26 @@ def move(self, keys):
         global LAST_POSITION
         LAST_POSITION = self.pos_x, self.pos_y
         if keys[pygame.K_LEFT]:
-            self.pos_x -= self.speed
+            if self.pos_x >= self.speed:
+                self.pos_x -= self.speed
+            else:
+                self.pos_x = self.pos_x
```

Comment:
oq é que vai acontece se no futuro a gente mudar o tamnho da tela, speed ou qualquer coisa e a posição for 3 e o speed for tipo 6?